### PR TITLE
add nginx container

### PR DESCRIPTION
**Pull request by byang-intel**
Signed-off-by: Bin Yang <bin.yang@intel.com>

### CODE REVIEW

File: nginx/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+RUN mkdir -p /var/lib/nginx/
+RUN mkdir -p /etc/nginx/conf.d
+RUN cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+RUN echo 'server {' > /etc/nginx/conf.d/default.conf
+RUN echo '  server_name localhost;' >> /etc/nginx/conf.d/default.conf
+RUN echo '  location / {' >> /etc/nginx/conf.d/default.conf
+RUN echo '    root /usr/share/nginx/html;' >> /etc/nginx/conf.d/default.conf
+RUN echo '    autoindex on;' >> /etc/nginx/conf.d/default.conf
+RUN echo '  }' >> /etc/nginx/conf.d/default.conf
+RUN echo '}' >> /etc/nginx/conf.d/default.conf
```
Comment (bryteise) [inline nginx/Dockerfile]: Is there a reason why we aren't just COPYing this file in? All these RUN commands create a new layer.

**Comment by bryteise** [comment]
Just have the one comment but otherwise this is looking good. Thanks!

File: nginx/Dockerfile (hunk 1 above)
Comment (byang-intel) [reply nginx/Dockerfile]: yes, it is better to use COPY here. I will update PR

**Comment by bryteise** [comment]
@gtkramer We'll need to add hooks for publishing this next.

**Pull request merged**